modules/admin_manual/pages/configuration/user/user_roles.adoc

```diff
@@ -21,6 +22,7 @@ The following information is not an in-depth guide, but more of a high-level ove
 
 == Guest
 
+* Only available after installing the {guests-app-url}[Guests app]
```

Comment:
```suggestion
* The {oc-marketplace-url}/apps/guests[Guests app] is available on the ownCloud Marketplace. You must install and enable it first.
```
The attribute version you do is possible and valid, but we do the style here which eases proofreading if the stuff gets more complex.

Reply:
I don't mind, I was just following how it was done for the other apps. I would recommend to change that as well then  ;)